cmd/cone/search_entitlements.go

```diff
@@ -48,6 +49,7 @@ func searchEntitlementsRun(cmd *cobra.Command, args []string) error {
 		EntitlementAlias: alias,
 		GrantedStatus:    grantedStatus,
 		AppDisplayName:   v.GetString(appDisplayNameFlag),
+		IncludeDeleted:   false,
```

Comment:
Should this be using the value from the flag, instead of always setting it to false?

Reply:
Good catch. Thanks.